scripts/integration-tests/e2e-babel-old-version.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+#==============================================================================#
+#                                  SETUP                                       #
+#==============================================================================#
+
+# Start in scripts/integration-tests/ even if run from root directory
+cd "$(dirname "$0")" || exit
+
+source utils/local-registry.sh
+source utils/cleanup.sh
+
+# Echo every command being executed
+set -x
+
+# Go to the root of the monorepo
+cd ../..
+
+#==============================================================================#
+#                                   TEST                                       #
+#==============================================================================#
+
+startLocalRegistry "$PWD"/scripts/integration-tests/verdaccio-config.yml
+yarn upgrade --scope @babel
```

Comment:
Is `yarn upgrade` working again in our repo?

Reply:
We only need to upgrade the dependencies of the top-level `package.json`, since they are what we use to compile.

Reply:
Oh ignore me, it doesn't work.

Reply:
We have replaced `yarn upgrade` by an ad-hoc script in #11021 due to https://github.com/yarnpkg/yarn/issues/7797. I am not sure if that issue has been fixed or it affects only `yarn upgrade-interactive` so `yarn upgrade` is good.

Reply:
It doesn't give problems here because we are not using yarn workspaces.